mappings/net/minecraft/block/entity/BlockEntity.mapping

```diff
@@ -42,6 +44,14 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 		ARG 2 state
 	METHOD method_31664 setCachedState (Lnet/minecraft/class_2680;)V
 		ARG 1 state
+	METHOD method_38235 toUpdatePacket ()Lnet/minecraft/class_2596;
+		COMMENT Implement and return a packet that should be sent to players nearby when the observable state of
+		COMMENT this block entity changes.
+		COMMENT <p/>
```

Comment:
```suggestion
		COMMENT <p>
```
`<p/>` is like `<p></p>`, I suppose? We want the text in the paragraph than separate them with paragraphs.